spring-cloud-gcp-autoconfigure/pom.xml

```diff
@@ -117,6 +117,18 @@
             <scope>test</scope>
         </dependency>
 
+        <!-- Java-CFEnv -->
+        <dependency>
+            <groupId>io.pivotal.cfenv</groupId>
+            <artifactId>java-cfenv</artifactId>
```

Comment:
This should be optional, and we should document that one must add this dependency if they're using CF, or maybe we need to create a starter for CF.